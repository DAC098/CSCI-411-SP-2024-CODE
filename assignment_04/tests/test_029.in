6
3 -3 3
gteawlywgagltotoneqmgqz gaailryakcdyyrocafnnabkoxkvo
ntvmggnktrfkursysgfs vnmgyrpswlrjjddceppdbtdx
gtkxgvblmydojm vugjisopngxfyvpkmucdttq
gctgggcttttggttc tccttagtgcatg
cagcgtattgtgtt aaaagaaa
tgtaaaggccccgttct taaaac
