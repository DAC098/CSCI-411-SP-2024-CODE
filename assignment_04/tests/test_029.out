___________________________gteawlywgagltotoneqmgqz
gaailryakcdyyrocafnnabkoxkv_______________o_______
15
________________________ntvmggnktrfkursysgfs
vnmgyrpswlrjjddceppdbtdx____________________
12
_______________________gtkxgvblmydojm
vugjisopngxfyvpkmucdttq______________
27
___________gctgggcttttggttc
tccttagtgca__________t_g___
-9
______cagcgtattgtgtt
aaaaga_a____a_______
-18
__tgtaaaggccccgttct
ta___aaa_________c_
-33
